usagov/usagov-2021
# Usagov 1241 mobile search bar

## Jira Task

https://cm-jira.usa.gov/browse/USAGOV-1241

## Description

Added the white background for the mobile search bar. Also moved the header sass out into it's own file. 

## Type of Changes

  - Add screenshot showing what it should look like

## Testing Instructions



### Requires New Config

### Requires New Content

### Validation Steps
- Open  xcode simulator if possible. (Background even without the changes looks white on native browsers - see the Jira ticket). 
- Use the xcode simulator to open localhost and the menu. Check if the background of the search bar is white or grey. It should be white. 

## Security Review




## Reviewer Reminders
- Reviewed code changes
- Reviewed functionality
- Security review complete or not required

## Post PR Approval Instructions
Follow these steps as soon as you merge the new changes.

1. Go to the [USAGov Circle CI project](https://app.circleci.com/pipelines/github/usagov/usagov-2021).
2. Find the commit of this pull request.
3. Build and deploy the changes.
4. Update the Jira ticket by changing the ticket status to `Review in Test` and add a comment. State whether the change is already visible on [cms-dev.usa.gov](http://cms-dev.usa.gov/) and [beta-dev.usa.gov](http://beta-dev.usa.gov/), or if the deployment is still in process.
